src/render-domain.ts

```diff
@@ -306,28 +307,37 @@ export default class MainDomain
   async computeGlobalStats(
     accountAddresses?: string[],
   ): Promise<Global${Name}Stats> {
-    const accountsStats = await this.getAccountStats(accountAddresses)
+    const accountsStats = await this.getAccountStats<TimeStats>(accountAddresses)
     const globalStats: Global${Name}Stats = this.getNewGlobalStats()
 
     for (const accountStats of accountsStats) {
       if (!accountStats.stats) continue
 
-      const { totalRequests, totalUniqueAccessingPrograms, totalAccounts } =
-      accountStats.stats
+      const { accesses, accessesByProgramId, startTimestamp, endTimestamp } =
+        accountStats.stats
 
       const type = this.discoverer.getAccountType(accountStats.account)
 
-      globalStats.totalAccounts[type] += totalAccounts[type]
-      globalStats.totalRequests += totalRequests
-      globalStats.totalUniqueAccessingPrograms += totalUniqueAccessingPrograms
+      globalStats.totalAccounts[type]++
+      globalStats.totalAccesses += accesses
+      Object.entries(accessesByProgramId).forEach(([programId, accesses]) => {
+        globalStats.totalAccessesByProgramId[programId] =
+          (globalStats.totalAccessesByProgramId[programId] || 0) + accesses
+      })
+      globalStats.startTimestamp = Math.min(
+        globalStats.startTimestamp || Infinity, startTimestamp || Infinity,
```

Comment:
```suggestion
        globalStats.startTimestamp || Number.MAX_SAFE_INTEGER, startTimestamp || Number.MAX_SAFE_INTEGER,
```
Needed to replace Infinity with MAX_SAFE_INTEGER